### ISSUE TITLE
InfluxDb 2 Uses Wrong Port By Default

### ISSUE THREAD

**Issue by mark-monteiro** [tiredofit/docker-db-backup#152]
### Summary

When running a backup for InfuxDb on version 2.x, if `DB_PORT` isn't specified then port 8088 is automatically selected, but it should be port 8086.

The backup process for Influx 1.x [uses the RPC port 8088](https://docs.influxdata.com/influxdb/v1.8/administration/backup_and_restore/#configure-backup-and-restore-services), but that is not the case for influx 2.x.

### Steps to reproduce

Set up a backup for an Influx 2.x instance without a value for `DB_PORT` specified. Connect to the image and run `backup-now`, see the following error:

```
** Performing Manual Backup
curl: (7) Failed to connect to scrutiny port 8088 after 0 ms: Connection refused
2022-09-11.22:34:21 [WARN] ** [db-backup] InfluxDB Host 'https://db' is not accessible, retrying.. (5 seconds so far)
```

### What is the expected *correct* behavior?

The default port selected is 8086 and the backup succeeds.

### Relevant logs and/or screenshots

See backup logs above

### Environment

- **Image version / tag:**  ghcr.io/analogj/scrutiny:v0.5.0-omnibus
- **Host OS:** Ubuntu

**Comment by tiredofit** [comment]
Hi there this has been fixed in version `3.4.0` . [Changelog](https://github.com/tiredofit/docker-db-backup/commit/dc4ab0bfc59f5c4580b2edc0ecfe19e57b168816) - Thanks for the submission, please let me know if this doesn't work out as intended.

**Comment by mark-monteiro** [comment]
Confirmed this appears to be working as intended now. Thanks again for the quick turnaround!
